inc/search.class.php

```diff
@@ -539,7 +539,7 @@ static function constructSQL(array &$data) {
 
       //// 1 - SELECT
       // request currentuser for SQL supervision, not displayed
-      $SELECT = "SELECT '".Toolbox::addslashes_deep($_SESSION['glpiname'])."' AS currentuser,
+      $SELECT = "SELECT DISTINCT `$itemtable`.`id` AS id, '".Toolbox::addslashes_deep($_SESSION['glpiname'])."' AS currentuser,
```

Comment:
Previous code was suggesting that `$itemtable` could be empty. Was it right ?

Reply:
As discussed on IRC with @orthagh ; I do not see any case this can be empty (how to search any type in glpi without its table?)

Reply:
ok